pkg/assessor/manifest/manifest.go

```diff
@@ -210,6 +227,38 @@ func useADDstatement(cmdSlices map[int][]string) bool {
 	return false
 }
 
+func sensitiveVars(cmd string) bool {
+	if !strings.Contains(cmd, "=") {
+		return false
+	}
+
+	for _, s := range sensitiveWords {
+		sensitiveWords = append(sensitiveWords, strings.ToUpper(s))
+	}
+	pat := strings.ReplaceAll(`.*(REP).*`, "REP", strings.Join(sensitiveWords, "|"))
```

Comment:
Instead of using `Replace` I think this a good opportunity for [fmt.Sprintf](https://pkg.go.dev/fmt#Sprintf):
```
pat := fmt.Sprintf(`.*(%s).*`, strings.Join(sensitiveWords, "|"))
```

Reply:
Now using Sprintf instead of ReplaceAll